src/particles/MultiPlasma.H

```diff
@@ -103,6 +103,9 @@ public:
      */
     void TileSort (amrex::Box bx, amrex::Geometry geom);
 
+    /** returns u_std of the plasma distribution */
+    amrex::RealVect get_u_std () const;
```

Comment:
For consistency, could you make this function name `CamelCase` rather than `snake_case`?